config/hiveadmission/hiveadmission_rbac_role.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
maybe for validating later, but I don't believe hiveadmission needs any permissions for dnszones. We are able to validate other resources (clusterdeployments, clusterimagesets) without explicit permissions to those.